providers/google.go

```diff
@@ -219,26 +230,60 @@ func (g *GoogleOp) requestTokens(ctx context.Context, cicHash string) ([]byte, e
 		if g.httpSessionHook != nil {
 			defer shutdownServer()
 		}
-		return nil, err
-	case token := <-ch:
-		return token, nil
+		return nil, nil, nil, err
+	case tokens := <-chTokens:
+		return []byte(tokens.IDToken), []byte(tokens.RefreshToken), []byte(tokens.AccessToken), nil
```

Comment:
There is no issue with doing this in memguard and earlier versions of this PR did what you propose.

https://github.com/openpubkey/openpubkey/blob/1670fea42151668fa320ae3f31b628f8cc46bf19/providers/google.go#L220-L225

It seems to add extra code with a slight increase in readability but more effort for implementers. I'm on the fence about it.

Reply:
Yeah, one plus for the struct and implementations is that you don't need to remember the order of the return values to determine which is the ID vs. refresh vs. access token []byte. It is a bit more explicit that way.

Reply:
Added a token struct. Looking at the code I'm still not convinced it is worth it over []byte, []byte, []byte. It does provide some additional safety and flexibility.

You still need to pass some of the values in the mocks as []byte, []byte, []byte because if you put the Tokens struct in the mock package you get circular dependencies between provider and provider.mocks.
https://github.com/openpubkey/openpubkey/pull/179/commits/f01f6c897c20976fecfb2c1918b9bf99427c1d1c

Reply:
After sleeping on this, I think I know how to make the Tokens struct work. I just need to move it into the oidc package, then all the mocks can use it and it simplifies code while increasing safety.

Reply:
Thanks @mrjoelkamp  @ymarcus93 for pushing me to make this token struct. I gave up too early when i tried this approach before, but trying it again I think I got it right. See commit here:

https://github.com/openpubkey/openpubkey/pull/179/commits/2e9ec9124e3df2e07da4b34d65bd443b3c3d297d